src/components/Schedule/schedule.tsx

```diff
@@ -0,0 +1,101 @@
+import { CalendarEvent, WeekDate } from "../../types"; 
```

Comment:
import { formatTime } from  'format-time' - my nie wiemy czy to biblioteka czy też folder

jeśli taka ścieżka to "../../types" - wady: jeśli przeniesiesz plik schedule to trzeba poprawić ścieżki 

import {formatTIme } from '@/format-time' -> src/format-time